bundled/tool/server.py

```diff
@@ -461,7 +461,7 @@ def _get_default_settings(workspace_path: str) -> Dict[str, str]:
         "logLevel": "error",
         "args": [],
         "severity": {
-            "E": "Warning",
+            "E": "Hint",
             "W": "Warning",
```

Comment:
Warnings in `isort` are often about outdated arguments or deprecations. These are not associated with a particular file. The issue was that improper import organization is treated as an error by `isort`, and users want the default level for that to be `Hint`. `isort` only has one error that it currently reports on imports which is about the improper import organization.